### PR TITLE
fix(DB/Creature) flags_extra for brewfest barker bunny's

### PR DESCRIPTION
**Pull request by locus313**
## CHANGES PROPOSED:
-  change flags_extra for brewfest barker bunny's so they are only visible to gm's

## ISSUES ADDRESSED:
- Closes none

## TESTS PERFORMED:
- apply sql and restart server
- checked that each brewfest barker bunny is only visible to gm's
 

## HOW TO TEST THE CHANGES:

- apply sql and restart server
- .go c 240003-240010
- .gm off (check that brewfest barker bunny is invisible)
- .gm on (check that brewfest barker bunny is visible)
## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by locus313** [comment]
Tested with gm on:
image

Tested with gm off:
image

**Review by Si1ker — APPROVED** [review]
A bunny? where's ma' infernal? 😹

**Comment by locus313** [comment]
Did some more testing and look like changing the phasemask breaks the Bark For quests. For example: 
https://www.wowhead.com/quest=11407/bark-for-drohns-distillery

File: data/sql/updates/pending_db_world/rev_1601091401144068714.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1601091401144068714');
+
+UPDATE `creature` SET `phaseMask` = 192 WHERE `id` IN (24202, 24203, 24204, 24205);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1601091401144068714.sql:3]:
```suggestion
UPDATE `creature_template` SET `flags_extra`=`flags_extra`|0x00000080 WHERE `entry` IN (24202, 24203, 24204, 24205);
```

**Review by Kitzunu — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1601091401144068714.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1601091401144068714');
+
+UPDATE `creature_template` SET `flags_extra`=`flags_extra`|0x00000080 WHERE `entry` IN (24202, 24203, 24204, 24205);
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1601091401144068714.sql:3]: cc @Si1ker

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by Racknar0** [comment]
For these small details I love azerothcore

Tested

before PR

image
image

after  PR

image
image

**Pull request merged**